src/main/resources/files/DuplicateSuperHeroException.java.tpl

```diff
@@ -0,0 +1,9 @@
+package [# th:text="${java_package}"/].graphql.db;
```

Comment:
Can you add copyright statement?

Reply:
Template files and generated code files do no need copyright statements (the other .tpl files doesn't have them either)